README.md

```diff
@@ -27,3 +26,9 @@ Make many small **Pull Requests** rather than packing too much code into one bra
 We want every prefab or asset to be able to be used in any scene. If there are steps needed to get it working (initializing variables, etc), document them here.
 
 We plan to demo all features weekly.
+
+## Documentation
+### Title Screen
+
+- GetVersion.cs - A script that pulls the current build version (We set that in Project Settings under Player tab)
+				and puts it into a transparent text in the bottom left corner. It can for example indicate which version is a screenshot from.
```

Comment:
There are a lot of binaries in my folders that were changed (the Magic Pack Demo). What happened with those assets that the binaries were changed?

Reply:
I don't have any idea, I didn't touch it and Unity did it by itself right after opening it... Maybe it was updated or something

Reply:
It's hard to tell if anything changed, on my end everything seems to work fine

Reply:
It seems that the same situation happened to Arthur, so Unity had to be the cause